packages/xo-server/src/api/sr.mjs

```diff
@@ -1015,14 +1019,23 @@ disableMaintenanceMode.resolve = {
 
 // -------------------------------------------------------------------
 
-export async function reclaimSpace({ sr }) {
+export async function reclaimSpace({ sr, bypassBackupCheck = false }) {
+  if (bypassBackupCheck) {
+    log.warn('sr.reclaimSpace with argument "bypassBackupCheck" set to true', { srId: sr.id })
+  } else {
+    await backupGuard.call(this, sr.$poolId)
```

Comment:
use `$pool` instead of `$poolId`. `$poolId` is deprecated. https://github.com/vatesfr/xen-orchestra/blob/master/%40vates/types/src/xo.mts#L27